supertokens_python/recipe/dashboard/exceptions.py

```diff
@@ -0,0 +1,5 @@
+from supertokens_python.exceptions import SuperTokensError
+
```

Comment:
is this file needed?

Reply:
`is_error_from_this_recipe_based_on_instance` in `recipe.py` uses it. However, we don't have any exceptions as of now.